lib/livebook/notebook/explore/intro_to_vega_lite.livemd

```diff
@@ -28,6 +28,68 @@ alias VegaLite, as: Vl
 
 <!-- livebook:{"branch_parent_index":0} -->
 
+## The Chart cell
+
+Before we get into exploring all the various chart types, let's have
+a look at an awesome feature that comes with `kino_vega_lite` - the
+**Chart** cell!
+
+Coding up a chart usually involves a couple steps. If you don't know
+the API of a particular library, it may be a bit challenging. On the
+other hand, if you know the API in-and-out, it's a rather repetitive
+task. That's where the **Chart** cell comes in, it is a special type
+of cell that magically writes all of the chart code for us. Let's
+give it a try!
+
+First, we need some data to work with, here's a small excerpt from
+the popular "Iris" dataset:
```

Comment:
Link to the dataset maybe?

Reply:
Good call, I added a link to the [dataset on kaggle](https://www.kaggle.com/datasets/uciml/iris).